cmd/geth/config.go

```diff
@@ -184,6 +184,11 @@ func makeFullNode(ctx *cli.Context) *node.Node {
 		cfg.Eth.Enable4844 = override4844
 	}
 
+	if ctx.IsSet(utils.OverrideOdysseyForkTimeFlag.Name) {
```

Comment:
We may want to have a name for the hard fork. Refer to the previous iliad geth hard fork [here](https://github.com/piplabs/story-geth/commit/a54ef9e719c7429ce70921a2296a9157a7c323dd#diff-efb34882aeb5938c3c2527733e914b31697acefcdccc3b3c42467fda64efc362R181)

Reply:
Good. I will change the name of hardfork when its name is determined.

Reply:
I think `theogony` makes sense as we discussed today and proposed by Tim